### ISSUE TITLE
Fine-Tuning on Schubert's lieder compositions

### ISSUE THREAD

**Issue by M1k3Dm** [ElectricAlexis/NotaGen#33]
Has anyone run the fine-tuning example on Schubert’s works as suggested in the README? I’m running with the large pre-trained model and the default config training parameters (batch size: 1, learning_rate: 1e-5, num_epochs: 64) on A40 (48GB), but I don’t observe any improvement as the epochs progress. Specifically, the best epoch is the 3rd (out of 64) with a minimum eval loss of 0.131. I want to perform fine-tuning on a separate dataset and I’m trying to figure out whether it’s my mistake or this is normal for this specific dataset.

**Comment by OpenSourceAnarchist** [comment]
I'm pretty sure this is related to my comment (and code change unfortunately) in #34 . If so reverting it to the `bbbdab4` commit before mine should work even if there is an issue with normalization in `utils.py`. I'm working on a fix for this that shouldn't introduce any other bugs. Truly sorry if this is the case and will let you know once I have a working branch. I also want to RL on a dataset so this is important to me to get working!
